rsa_pss.go

```diff
@@ -15,7 +15,7 @@ type SigningMethodRSAPSS struct {
 	// VerifyOptions is optional. If set overrides Options for rsa.VerifyPPS.
 	// Used to accept tokens signed with rsa.PSSSaltLengthAuto, what doesn't follow
 	// https://tools.ietf.org/html/rfc7518#section-3.5 but was used previously.
-	// See https://github.com/dgrijalva/jwt-go/issues/285#issuecomment-437451244 for details.
+	// See https://github.com/golang-jwt/jwt/issues/285#issuecomment-437451244 for details.
```

Comment:
Please refrain from just blindly doing a `s/dgrijalva/jwt-go/golang-jwt/jwt`. Until we know if we can transfer the old repositories, links to issues still need to refer to the old repo.

Reply:
sorry i blindly changed those without knowing.